### ISSUE TITLE
tW word count accuracy

### ISSUE THREAD

**Issue by mandolyte** [unfoldingWord-box3/book-package#14]

**Comment by mandolyte** [comment]
Component counts match test results:
```
$ nx apostle.mjs
(node:2204) ExperimentalWarning: The ESM module loader is experimental.
Passed: apostle.txt  Total: 279  Distinct: 134
$
```

**Comment by mandolyte** [comment]
Here is what the component is showing:
```
Translation Words for "TIT" and Chapters 1
Distinct Number of tW Articles: 71
Total Number of tW Articles: 99

Distinct Number of tW article words: 3287
Total Number of tW article words: 22103
```

I will close this as verified based on the following reasoning:
- I was able to copy/paste the details table into a spreadsheet (see `src/tests/tW/tit_ch1.xlsx`). From there I verified the number of tW articles at 71; the total number of words in those 71 articles (22103).
- The simple sum total of distinct words is 10464, which is 3x higher than the number above. This feels reasonable, so I'm giving it a pass.
